libs/common/backend/models/src/interfaces/app.interface.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * The App configuration interface
+ */
+export interface IAppConfig {
+  /**
+   * The port the application will run on
+   * @type {number}
+   */
+  port: number;
+
+  /**
+   * The host the application will run on
+   * @type {string}
+   */
+  host: string;
+
+  /**
+   * The name of the application
+   * @type {string}
```

Comment:
I like that you always anotate the type, but I guess the IDE does this automatically (just a remark not need for changing it)